network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -572,13 +574,6 @@ def serve(self, request, *args, **kwargs):
 
         return super().serve(request, *args, **kwargs)
 
-    def save(self, *args, **kwargs):
-        # When a new ProductPage is created, ensure a vote bin always exists.
-        # We can use save() or a post-save Wagtail hook.
-        save = super().save(*args, **kwargs)
-        self.get_or_create_votes()
-        return save
-
```

Comment:
Why? Where do we now make sure that `self.votes` always exists?

Reply:
I ran into some "magic" yesterday. Looks like this is creating a one to one relationship automatically. This was causing a lot of "Votes already exists" validation errors in the migration.

Reply:
Re-added after using a ForeignKey